data/spells/scripts/monster/burning_gladiator_firering.lua

```diff
@@ -0,0 +1,16 @@
+area = {
```

Comment:
all monsters ring and X spells should be done in XML using what was implemented here: https://github.com/otland/forgottenserver/pull/3691